src/app/frames/nominate/route.tsx

```diff
@@ -2,26 +2,27 @@ import { Button } from "frames.js/next";
 import { getNominationsFromUserToday } from "@/app/_api/data/nominations";
 import { getUserBalances } from "@/app/_api/data/users";
 import { getWalletFromExternal } from "@/app/_api/data/wallets";
+import { searchBuilders } from "@/app/_api/functions/search-builders";
 import { frames, getFramesUser } from "@/app/frames/frames";
+import { BadRequestError } from "@/shared/utils/error";
 
 const handler = frames(async (ctx) => {
+  if (!ctx.message?.isValid) {
+    throw new BadRequestError("Invalid message");
```

Comment:
Is the frames handler able to handle this error? 
Having it inside the try block would shouw the "invalid message" with the template below.